### ISSUE TITLE
Option not to "print" the result of `importSchema`.

### ISSUE THREAD

**Issue by thealjey** [ardatan/graphql-import#355]
I would like to get the SchemaDocument ast from `importSchema` instead of the printed SDL.
Seems like a waste to have to parse it right after it was printed.

**Comment by ardatan** [comment]
Hi @ralphchristianeclipse  !
In 1.0.0 beta release, we introduced a lot of changes including this feature;
```ts
const returnValue = await importSchema(pointer, {}, 'DocumentNode');
```
Could you install `graphql-import@beta` to try new changes? Don't forget to modify your code regarding to the migration notes in `README`. 
https://github.com/ardatan/graphql-import#updating-from-07x

**Comment by ardatan** [comment]
Available in 1.0.0!
